apis/r/src/Makevars.in

```diff
@@ -10,7 +10,7 @@ all: $(SHLIB)
         # On macOS aka Darwin we call install_name_tool
         # Case one: If we had a downloaded TileDB Core artifact, adjust zlib path and add to @rpath
         # Case two: If we see the system libraries (on macOS) ensure /usr/local/lib rpath is considered
-	if [ `uname -s` = 'Darwin' ] && [ -f tiledbsoma.so ]; then \
+	@if [ `uname -s` = 'Darwin' ] && [ -f tiledbsoma.so ]; then \
```

Comment:
Honestly i prefer for this to be visible. It doesn't take up much space, and it's reassuring to see the output when debugging installs.

Reply:
But no other R package does it and compactness _is_ nicer IMHO and reduces cognitive load.  It really is an internal step just `configure`.  I could settle for a one line echo indicating that dynamic paths are updated.  Better?

For comparison, tiledb-r below.

<details>

```sh
** libs
using C compiler: ‘gcc-12 (Ubuntu 12.2.0-3ubuntu1) 12.2.0’
using C++ compiler: ‘g++-12 (Ubuntu 12.2.0-3ubuntu1) 12.2.0’
using C++17
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c RcppExports.cpp -o RcppExports.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c arrowio.cpp -o arrowio.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c batched.cpp -o batched.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c column_buffer.cpp -o column_buffer.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c deprecation.cpp -o deprecation.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c durations.cpp -o durations.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c libtiledb.cpp -o libtiledb.o
ccache gcc-12 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe             -c nanoarrow.c -o nanoarrow.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c nullable.cpp -o nullable.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c shmem.cpp -o shmem.o
ccache g++-12  -std=gnu++17 -I"/usr/share/R/include" -DNDEBUG -I. -I../inst/include/  -I/usr/local/include  -I'/usr/local/lib/R/site-library/Rcpp/include'     -fpic  -g -O3 -Wall -pipe -pedantic   -c utilities.cpp -o utilities.o
ccache g++-12 -std=gnu++17 -Wl,-S -shared -L/usr/lib/R/lib -Wl,-Bsymbolic-functions -flto=auto -ffat-lto-objects -flto=auto -Wl,-z,relro -o tiledb.so RcppExports.o arrowio.o batched.o column_buffer.o deprecation.o durations.o libtiledb.o nanoarrow.o nullable.o shmem.o utilities.o -L/usr/local/lib -ltiledb -L/usr/lib/R/lib -lR
installing to /usr/local/lib/R/site-library/00LOCK-tiledb-r/00new/tiledb/libs
** R
```

</details>